libs/blocks/global-navigation/global-navigation.js

```diff
@@ -35,6 +36,14 @@ const CONFIG = {
     loadDelayed: 2000,
     keyboardNav: 8000,
   },
+  features: [
```

Comment:
Should we put those into `utilities.selectors` ? Wouldn't mind having a nested object, as long as we have all of the selectors in a central place.

Reply:
These features only refer to the initial raw HTML, we're using them just to know how to decorate a section, they have no use beyond this. It's basically a list of features supported by the global-navigation.

The selectors we have in our files, however, refer to the already decorated elements. So to me, the two have different purposes, that's why I didn't really want to mix them. Currently, their main purpose is [this](https://github.com/adobecom/milo/pull/745/files#diff-628bf5a25903978c5b36da4943ec813da171346d24bd64f80907f012a840b098R497), to ensure we don't decorate an element as a main navigation item if it is nested inside another known feature.

I could be persuaded to move them over, if you think the different nature of their purpose can co-exist and might bring a benefit for the future.

Reply:
Oh yes, it's `gnav-brand` instead of `feds-brand` - you're correct.
In this case mixing them would lead to more confusion than help. Thanks for explaining, let's keep it as is!